src/app.ts

```diff
@@ -1,6 +1,22 @@
 
 const startApp = async () => {
-    
+    const list = await fetch('https://randomapi.com/api/8csrgnjw?key=LEIX-GF3O-AG7I-6J84')
+    .then(
+        result => result.json()
+    )
+    console.log({list});
 };
 
+const fetchList = async ( page) => {
+    const list = await fetch('https://randomapi.com/api/8csrgnjw?key=LEIX-GF3O-AG7I-6J84')
+    .then(
+        result => result.json()
+    )
+    return {list, page}
+}
+
+export function laodNext () {
+    console.log('isLoading');
+    return 'done '
```

Comment:
Codacy has a fix for the issue: [Replace `····return·'done·'` with `··return·"done·";`](https://app.codacy.com/gh/ZeeUnltd/starter/pullRequest?prid=10061291)
```suggestion
  return "done ";
```